Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -171,14 +171,11 @@ public function nodeType($nodeType)
      */
     public function sortDesc($propertyName)
     {
-        if (!isset($this->request['sort'])) {
-            $this->request['sort'] = array();
-        }
-
-        // http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/search-request-sort.html
-        $this->request['sort'][] = array(
+        $configuration = [
             $propertyName => array('order' => 'desc')
```

Comment:
Well, we should go through all of the code at all in another PR to refactor all arrays or not?